test/Interop/Cxx/operators/Inputs/member-out-of-line.h

```diff
@@ -4,6 +4,9 @@
 struct IntBox {
```

Comment:
Not you're fault, but we should change this name. Want to make a nfc patch to do this? If not, I'm happy to.

Reply:
I'll do it :)

Reply:
Great, thank you!